test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -89,12 +99,13 @@ baz
 "@
         $expected = 'foo'
         $tailCount = 3
+        [Microsoft.PowerShell.Commands.FileSystemCmdletProviderEncoding] $encoding = $EncodingName
```

Comment:
This line is unncessary, the string will be converted to the correct enum value when passed as parameter below.  So you can just change the test case parameter name above to just `Encoding`

Reply:
I know but I added it for two reasons...

1: I was mistakenly trying to use values from Text.Encoding.GetEncodings() so I want it to be clear that it is not a standard Encoding value.
2: I want to have it to be easy to find if the Jim's Encoding refactoring is done.